tests/serde.rs

```diff
@@ -0,0 +1,34 @@
+use std::{collections::HashMap, fs::File, path::Path};
+
+use gvas::GvasFile;
+use serde_json::json;
+
+fn get_hints() -> HashMap<String, String> {
+    let mut hints = HashMap::new();
+
+    hints.insert(
+        "MinersManualKnownObjects.SetProperty.StructProperty".to_string(),
+        "Struct".to_string(),
+    );
+    hints.insert(
+        "GameplayDatabase.MapProperty.Value.StructProperty".to_string(),
+        "Struct".to_string(),
+    );
+    hints.insert(
+        "PlayerAttributes.MapProperty.Key.StructProperty".to_string(),
+        "Struct".to_string(),
+    );
+
+    hints
+}
+
+#[test]
```

Comment:
Instead of making serde enabled by default, you could use `#[cfg(feature = "serde")]` to guard this test.

Reply:
The related `use` caused the CI failure; you can fix that by moving the use into a block that is guarded by the suggestion above